src/wallet/feebumper.cpp

```diff
@@ -57,6 +57,58 @@ static feebumper::Result PreconditionChecks(interfaces::Chain::Lock& locked_chai
     return feebumper::Result::OK;
 }
 
+//! Check if the user provided a valid feeRate 
+static feebumper::Result CheckFeeRate(const CWallet* wallet, const CWalletTx& wtx, const CFeeRate& newFeerate, const int64_t maxTxSize, std::vector<std::string>& errors) {
+    // check that fee rate is higher than mempool's minimum fee
```

Comment:
is this block of code cribbed from somewhere to compare with?

Reply:
Yeah, this code is from the `CreateTotalBumpTransaction` method https://github.com/bitcoin/bitcoin/blob/b89249e0de8ad1848d487dd3e7a85a0d6d456de8/src/wallet/feebumper.cpp#L199-L247